dialogue-core/src/main/java/com/palantir/dialogue/core/AimdConcurrencyLimiter.java

```diff
@@ -106,7 +110,10 @@ void ignore() {
          */
         void dropped() {
             inFlight.decrementAndGet();
-            limit.accumulateAndGet(inFlightSnapshot, LimitUpdater.DROPPED);
+            int newLimit = limit.accumulateAndGet(inFlightSnapshot, LimitUpdater.DROPPED);
+            if (log.isDebugEnabled()) {
+                log.debug("DOWN {}", SafeArg.of("newLimit", newLimit));
```

Comment:
I really wish we could use timber here since it does the defensive level check for you

Reply:
Honestly I've actually acclimatised to this `if` pattern in perf-sensitive library code (obviously don't want this all over normal services, but kinda non painful)

Reply:
eh I think it just adds a bunch of noise to the code